docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
+# output
 
 ```
+Now, we define the particle spacing, which is our numerical resolution.
+We also set the number of boundary layers, which need to be sufficiently
+large, depending on the smoothing kernel and smoothing length, so that
+the compact support of the smoothing kernel is fully sampled with particles
+for a fluid particle close to a boundary.
+```jldoctest tut_setup; output = false
+fluid_particle_spacing = 0.05
+boundary_layers = 3
 
+# output
+3
+```
+
+## Experiment setup
+
+We want to simulate a water column resting under hydrostatic pressure inside
+a rectangular tank.
+First, we define the physical parameters gravitational acceleration,
+initial fluid size, tank size, fluid density, and simulation time.
+```jldoctest tut_setup; output = false
+gravity = 9.81
+tspan = (0.0, 1.0)
+initial_fluid_size = (1.0, 0.9)
+tank_size = (1.0, 1.0)
+fluid_density = 1000.0
+
+# output
+1000.0
+```
+
+In order to have the initial particle mass and density correspond to the
+hydrostatic pressure gradient, we need to define a state equation, which
+relates the fluid density to pressure.
+Note that we could also skip this part here and define the state equation
+later when we define the fluid system, but then the fluid would be initialized
+with constant density, which would cause it to oscillate under gravity.
+```jldoctest tut_setup; output = false
+sound_speed = 10.0
+state_equation = StateEquationCole(; sound_speed, reference_density=fluid_density,
+                                   exponent=7)
+
+# output
+StateEquationCole{Float64, false}(10.0, 7.0, 1000.0, 0.0)
+```
+The speed of sound here is numerical and not physical.
+We artificially lower the speed of sound, since the physical speed of sound
+in water would lead to prohibitively small time steps.
+The speed of sound in Weakly Compressible SPH should be chosen as small as
+possible for numerical efficiency, but large enough to limit density fluctuations
+to about 1%.
+
+TrixiParticles.jl requires the initial particle positions and quantities in
+form of an [`InitialCondition`](@ref).
+Instead of manually defining particle positions, you can work with our
+pre-defined setups.
+Among others, we provide setups for rectangular shapes, circles, and spheres.
+Initial conditions can also be combined with common set operations.
+See [this page](@ref initial_condition) for a list of pre-defined setups
+and details on set operations on initial conditions.
+
+Here, we use the [`RectangularTank`](@ref) setup, which generates a rectangular
+fluid inside a rectangular tank, and supports a hydrostatic pressure gradient
+by passing a gravitational acceleration and a state equation (see above).
+```jldoctest tut_setup; output = false, filter = r"RectangularTank.*"
+tank = RectangularTank(fluid_particle_spacing, initial_fluid_size, tank_size,
+                       fluid_density, n_layers=boundary_layers,
+                       acceleration=(0.0, -gravity), state_equation=state_equation)
```

Comment:
do we have plot recipes for IC? It would be cool to add here a line 
`plot(tank)` which plots the tank with the initial density.

Reply:
Well, it will be almost the same as the final plot, since it's a static simulation. We could also change this tutorial to use a dam break.